index.html

```diff
@@ -0,0 +1,282 @@
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <ul class="social-links">
+                    <li>
+                        <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedinicon.svg" alt="LinkedIn"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector (1).svg" alt="Angel List"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/github.svg" alt="Github"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vector.svg" alt="Meduim"></a>
+                    </li>
+                </ul>
+
+            </div>
+            <button id="normal_button1">
+                <p id="resume">Get my resume</p>
+            </button>
+            <ul id="skill_list">
+                <li id="languages">
+
+                    <div id="langbox">
+                        <p id="lang_text">Languages</p>
+                        <img id="down_arrow" src="images/downarrow.svg">
+                    </div>
+                    <ul>
+                        <li><a id="langicon" href="#"><img src="images/javascript.svg" alt="javascript"></a></li>
+                        <li><a id="langicon" href="#"><img src="images/html.svg" alt="html"></a></li>
+                        <li><a id="langicon" href="#"><img src="images/css.svg" alt="css"></a></li>
+                    </ul>
+                </li>
+                <li id="frameworks">
+                    <div id="framebox">
+                        <p id="frame_text">Frameworks</p>
+                        <img id="right_arrow1" src="images/rightarrow.svg">
+                    </div>
+                </li>
+                <li id="skills">
+                    <div id="skillsbox">
+                        <p id="skills_text">Skills</p>
+                        <img id="right_arrow2" src="images/rightarrow.svg">
+                    </div>
+                </li>
+            </ul>
+        </section>
+
+        <section id="contact_me">
+            <div class="contact_me_bg">
+                <div class="shapes">
+                    <p id="contact_me_text">Contact me</p>
+                    <p id="sub_contact">If you have an application you are interesed in developing, a feature that you
+                        need
+                        built or a project that needs coding. I'd love to help with it.</p>
+                    <form name="frmDataEntry" action="default.html" method="get">
+                        <input class="username" type="text" placeholder="Name and surname" maxlength="30"
+                            required></input>
+                        <input class="user_email" type="email" placeholder="beautifulemail@gmail.com" required></input>
```

Comment:
Hi @Alfredbis29 

I was just wondering about this part. You said my email should be equal to email. If I understand you correctly then my code is correct. My message is equal to text so that part I understand 100% why it's wrong. I would just like to know if I am missing something with the email part.